packages/settings-library/tests/test_base.py

```diff
@@ -16,10 +16,7 @@
 from settings_library.base import BaseCustomSettings
 
 
-@pytest.mark.parametrize(
-    "env_file",
-    (".env-compact", ".env-granular", ".env-fails", ".env-mixed", ".env-sample"),
-)
+@pytest.mark.parametrize("env_file", (".env-sample", ".env-fails"))
```

Comment:
Because I am dropping support for the `compact` format, these will get removed.
I've also added notes and a test to ensure support for `compact` is not present and instructions to put these files back if we decide to implement it in the future.